README.md

```diff
@@ -1,2 +1,12 @@
 # alman-akka
+
 :rainbow-flag: event calendar
+
+## web
```

Comment:
I would actually move the contents of the `web` folder to the root and make it as the "main" project.

Reply:
The reason I set it in an own folder was that the backend/db scripts looked like they also were in a subfolder? I can make this change but just wanna check that that is ok - like is it inconvenient to have to somehow ignore the db-scripts when running the web parts?

Reply:
Yes, let's make this as the root project and I will remove the db-scripts project later on. If the db scripts are not included to any app sources (./pages/*), they won't mess up web app building 🙂 To make things even more clearer, I could rename the db folder to tools - to make sure that peeps understand that it is only for dev-time activities.